Router/HostnameIdentifiedLoader.php

```diff
@@ -28,7 +28,7 @@ public function __construct(IdentifierMapping $identifierMapping)
         $this->identifierMapping = $identifierMapping;
     }
 
-    public function load($resource, $type = null): RouteCollection
+    public function load($resource, string $type = null): RouteCollection
```

Comment:
Volgens de docs bij `resourceStack` is een `resource` een string dus dat zou je hier wellicht ook kunnen hinten als dat klopt.